warc_metadata_sidecar.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/python
+
+__version__ = '1.0'
```

Comment:
It seems odd to me for this to be above imports. Is this something you have seen elsewhere?

Reply:
I agree it looks strange up there but evidently it is in the bible. - 

https://peps.python.org/pep-0008/#module-level-dunder-names

Reply:
I saw it in the "bible". 🙂